app/validators/questionnaire_schema.py

```diff
@@ -442,7 +454,11 @@ def _get_numeric_value(self, defined_value, system_default, answer_ranges):
         return system_default
 
     @staticmethod
-    def get_calculated_answer_ids(block):
+    def get_calculation_block_ids(block, source_type: str):
```

Comment:
Not a big problem but shouldn't we provide both type hints here?

Reply:
the majority of the file isn't type hinted so I wasn't sure how much value it would add but no harm in adding them - updated 👍